rotkehlchen/data_import/utils.py

```diff
@@ -0,0 +1,101 @@
+from abc import ABCMeta, abstractmethod
+from pathlib import Path
+from typing import Any, List, Tuple
+
+from rotkehlchen.accounting.ledger_actions import LedgerAction
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.db.ledger_actions import DBLedgerActions
+from rotkehlchen.exchanges.data_structures import AssetMovement, Trade
+from rotkehlchen.types import Location
+
+ROWS_PER_CONTEXT_SWITCH = 500
+
+TRADES_PER_DB_WRITE = 100
+ASSET_MOVEMENTS_PER_DB_WRITE = 100
+LEDGER_ACTIONS_PER_DB_WRITE = 100
+
+
+class BaseImporter(metaclass=ABCMeta):
```

Comment:
Question: Isn't `BaseImporter` quite vague? Maybe `BaseExchangeImporter`

Reply:
I might be wrong though but won't that conflict with `DataImporter` class?
One would expect that `DataImporter` should be a subclass of `BaseDataImporter`. Asides that, `BaseDataImporter` is fine